src/Razor/Razor.Design/test/IntegrationTests/Microsoft.AspNetCore.Razor.Design.Test.csproj

```diff
@@ -35,6 +35,27 @@
     <Reference Include="Microsoft.AspNetCore.Razor.Test.MvcShim.ClassLib" />
   </ItemGroup>
 
+  <Target Name="GenerateBuildVariablesFile" Condition="'$(MSBuildRuntimeType)' == 'Full'" BeforeTargets="BeforeBuild">
```

Comment:
This would be problematic on Windows if you do `dotnet test` no?

Reply:
Isn't it the same as before though? We had to run `build.cmd /t:Prepare` before running `dotnet test`. We compile include the generated file if it exists in all cases.

Reply:
Spoke offline. Added a warning for cases when someone runs `dotnet test` without building with desktop MSBuild first.